python/ray/serve/utils.py

```diff
@@ -431,3 +434,65 @@ def build_starlette_response(self) -> starlette.responses.Response:
             b"".join(self.buffer),
             status_code=self.status_code,
             headers=dict(self.header))
+
+
+def make_fastapi_class_based_view(fastapi_app: FastAPI, cls: Type) -> None:
```

Comment:
I don't fully understand this logic, is it possible to add some unit tests for it?